web_poet/exceptions.py

```diff
@@ -0,0 +1,13 @@
+class ReloadResponseData(Exception):
```

Comment:
I was thinking of a way to indicate the maximum retry attempts in this exception.

However, it's best to leave that to the framework using `web-poet` to standardize limits, delays, etc.